src/ert/gui/tools/manage_experiments/ensemble_init_configuration.py

```diff
@@ -95,51 +65,72 @@ def _addCreateNewEnsembleTab(self) -> None:
 
         self.addTab(panel, "Create new experiment")
 
-    def _addInitializeFromScratchTab(self) -> None:
+    def _add_initialize_from_scratch_tab(self) -> None:
         panel = QWidget()
         panel.setObjectName("initialize_from_scratch_panel")
-        layout = QVBoxLayout()
+        main_layout = QVBoxLayout()
+
+        ensemble_layout = QHBoxLayout()
+        ensemble_label = QLabel("Target ensemble:")
+        ensemble_selector = EnsembleSelector(self.notifier, show_only_undefined=True)
+        ensemble_selector.setMinimumWidth(300)
+        ensemble_layout.addWidget(ensemble_label)
+        ensemble_layout.addWidget(ensemble_selector)
+        ensemble_layout.addStretch(1)
 
-        target_ensemble = EnsembleSelector(self.notifier, show_only_undefined=True)
-        row = createRow(QLabel("Target ensemble:"), target_ensemble)
-        layout.addLayout(row)
+        main_layout.addLayout(ensemble_layout)
 
-        check_list_layout, parameter_model, members_model = createCheckLists(
-            self.ert_config.model_config.num_realizations,
-            self.ert_config.ensemble_config.parameters,
+        center_layout = QHBoxLayout()
+
+        parameter_model = SelectableListModel(
+            self.ert_config.ensemble_config.parameters
+        )
+        parameter_check_list = CheckList(parameter_model, "Parameters")
+        parameter_check_list.setMinimumWidth(500)
+        center_layout.addWidget(parameter_check_list)
+
+        members_model = SelectableListModel(
+            [
+                str(member)
+                for member in range(self.ert_config.model_config.num_realizations)
+            ]
         )
```

Comment:
This can be made somewhat more compact, but this is a style preference I guess;
```
        num_realizations = self.ert_config.model_config.num_realizations
        members_model = SelectableListModel(
            [str(member) for member in range(num_realizations)]
        )
```